packages/gestalt/src/FormElement.css

```diff
@@ -0,0 +1,33 @@
+.base {
```

Comment:
Since these are common across SelectList / TextField and TextArea, I'm extracting these out.